PIFMSMain/Market/Procedures/usp_GetMarketProfile.sql

```diff
@@ -0,0 +1,37 @@
+﻿/*
+This procedure gets Market profile based on the address1 passed.
+If address1 is not passed then returns all market profiles.
+*/
+CREATE PROCEDURE [Markets].[usp_GetMarketProfile]
```

Comment:
select all not required. please remove condition, and use input params wisely, get profile by market id.